website_team/__openerp__.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+{
+    "name": """Website Team""",
+    "summary": """Team""",
+    "category": "website",
+    "images": [
+        "static/src/img/ava_lack.png"
```

Comment:
images attribute is used at app store, which need to be something different. You don't need to register all your images here